src/psioptionseditor.cpp

```diff
@@ -260,7 +269,7 @@ void PsiOptionsEditor::tv_edit(const QModelIndex &idx)
 #if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
     } else if (value.type() == QVariant::Color) {
 #else
-    } else if (value.typeId() == QMetaType::QColor) {
```

Comment:
Please make sure the file is formatted with clang-format-18.
You can setup pre-commit.com and also setup proper clang-format version to make sure the file is properly formatted on commit.